pt_os_web_portal/miniscreen_onboarding_assistant/page_manager.py

```diff
@@ -83,6 +86,7 @@ def set_current_page_to(self, page):
 
         PTLogger.info(f"Page index: {self.current_page_index} -> {new_page_index}")
         self.current_page_index = new_page_index
+        state.set("miniscreen_onboarding", "state", str(self.current_page_index))
```

Comment:
This would cause the page to be reused on system reboot right? We should be saving the state _just_ before we do the system restart...